apps/question/views.py

```diff
@@ -67,3 +69,24 @@ def post(self, request, **kwargs):
       aa.answers.add(Answer.objects.get(id=answer_id))
 
     return Response(status=status.HTTP_201_CREATED)
+
+
+class QuestionAnswerView(APIAttemptBase):
+
+  def get(self, request, **kwargs):
+    """Returns answered question and picked answers from AttemptAnswer."""
+
+    attempt = self.attempt
+    question = Question.objects.get(id=self.kwargs['question_id'])
+
+    if question not in attempt.questions.all():
+      raise Http404
+
+    answer_attempt = AnswerAttempt.objects.get(attempt=attempt.id,
```

Comment:
What about checking attempt existence and user permissions to access that attempt?

Reply:
I already check that in parent class dispatch() when I get self.attempt.